protocol-impl/src/main/java/io/zeebe/protocol/impl/encoding/MsgPackConverter.java

```diff
@@ -144,39 +142,39 @@ private static void convert(
   ///////////////////////////////////// MSGPACK to MAP ///////////////////////////////////////////
   ////////////////////////////////////////////////////////////////////////////////////////////////
 
-  public static Map<String, Object> convertToMap(DirectBuffer buffer) {
+  public static Map<String, Object> convertToMap(final DirectBuffer buffer) {
     return convertToMap(OBJECT_MAP_TYPE_REFERENCE, buffer);
   }
 
-  public static Map<String, String> convertToStringMap(DirectBuffer buffer) {
+  public static Map<String, String> convertToStringMap(final DirectBuffer buffer) {
     return convertToMap(STRING_MAP_TYPE_REFERENCE, buffer);
   }
 
   private static <T extends Object> Map<String, T> convertToMap(
-      TypeReference<HashMap<String, T>> typeRef, DirectBuffer buffer) {
+      final TypeReference<HashMap<String, T>> typeRef, final DirectBuffer buffer) {
     final byte[] msgpackBytes = BufferUtil.bufferAsArray(buffer);
 
     try {
       return MESSSAGE_PACK_OBJECT_MAPPER.readValue(msgpackBytes, typeRef);
-    } catch (IOException e) {
+    } catch (final IOException e) {
       throw new RuntimeException("Failed to deserialize MessagePack to Map", e);
     }
   }
 
-  public static byte[] convertToMsgPack(Object value) {
+  public static byte[] convertToMsgPack(final Object value) {
```

Comment:
Could you please avoid doing this. I already discussed this with @npepinpe this is just noise. Either we decided that everything should be final (which I'm ok with) then we should do use all the same setting and do it once.

Reply:
Agree. Then, I'm voting for using **final** ;)

Reply:
ok I added it to the team meeting agenda